Reframe computational screens and tie limitations together across sections (#1007)
greenelab/covid19-review: content/20.pharmaceuticals.md

The difference between the preliminary evidence for fluvoxamine compared to other drugs that interact with Sigma receptors underscores a major critique of hypothesis-free screening: while these approaches allow for brute force comparison of a large number of compounds against a virus of interest, they lose the element of expertise that is associated with most successes in drug repurposing [@doi:10.1021/acs.jcim.0c00861].
Computational analysis correctly identified the Sigma-1 receptor as a protein of interest, but the process of identifying which drugs might modulate the interaction led to an emphasis on candidates that ultimately have not been supported. 
In comparison, computational techniques were used to design compounds targeting M^Pro^ based on an understanding of how this protease aids viral replication, and M^Pro^ inhibitors remain promising candidates [@doi:10.1039/D0CS01065K], although the clinical trial data is not yet available.
Despite the concerns it raises about high-throughput screening, it was a data-driven approach that allowed researchers to notice a shared property among many of the hits from computational screening analyses [@doi:10.1126/science.abi4708].

> In combination, these findings suggest that data-driven research can be an important component of the drug repurposing ecosystem, but that drug repurposing efforts that proceed without a hypothesis or without an emphasis on biological mechanisms are not likely to produce valuable candidates.

Reviewer: This is pretty strong ("are not likely"). The evidence looks that way... but perhaps different screens in the future will be more valuable? I wonder if there's a way to lean on the sentence that follows. Like, what if it ended with "may not produce valuable candidates."

Author: ```suggestion
In combination, these findings suggest that data-driven research can be an important component of the drug repurposing ecosystem, but that drug repurposing efforts that proceed without a hypothesis or without an emphasis on biological mechanisms may not produce valuable candidates.
```
Thanks @cgreene!
